docs/application/dotnet/samples/multimedia/overview.md

```diff
@@ -22,13 +23,13 @@
 			</td>
 			<td>
 			<p><strong>(W) ImageReader</strong> [In progress]</p>
-			<p>The ImageReader application demonstrates how to use image in Tizen .Net application.
-			This app is based on <a href="https://developer.xamarin.com/samples/xamarin-forms/WorkingWithImages/" target="_blank">xamarin sample app</a> and is modified according to Tizen Wearable UI.</p>
-			<p>You can get details from <a href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/images" target="_blank">Xamarin.Forms.Images</a></p><br>
+			<p>The ImageReader application demonstrates how to use an image in the Tizen .Net application.
+			This application is based on the <a href="https://developer.xamarin.com/samples/xamarin-forms/WorkingWithImages/" target="_blank">Xamarin sample application</a> and is modified according to the Tizen Wearable UI.</p>
+			<p>For more information, see <a href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/images" target="_blank">Xamarin.Forms.Images</a>.</p><br>
 			<p><b>Prerequisites</b><br>
 			Network connection is required for DownloadImage.</p>
-			<p>To use network connection in application,<br>
-			the following privileges are necessary:<br>
+			<p>To use network connection in an application,<br>
+			you must have the following privileges:<br>
 			<ul>
 				<li>http://tizen.org/privilege/internet</li>
```

Comment:
As this `http://tizen.org/privilege/internet` and `http://tizen.org/privilege/network.get` are not urls, how about applying code tag (\`) ?

Reply:
Yes, we can apply code tag. Sorry, I missed it.

Reply:
done